src/components/Login.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
```

Comment:
Let's use component directory scarcely. In each route, you can have a component directory where components belonging to that route can reside.

Reply:
Fixed